cmake/custom/fetch_mrcpp.cmake

```diff
@@ -18,7 +18,7 @@ else()
     GIT_REPOSITORY
       https://github.com/MRChemSoft/mrcpp.git
     GIT_TAG
-      5320eb433dcea82b177f5181ce12ab89f0b04a92
+      master
```

Comment:
Can you use a git tag. Master changes, so we need to be sure changes in master does not break vampyr